aws/resource_aws_lambda_function_test.go

```diff
@@ -666,8 +666,11 @@ func TestAccAWSLambdaFunction_FileSystemConfig(t *testing.T) {
 }
 
 func TestAccAWSLambdaFunction_tracingConfig(t *testing.T) {
-	var conf lambda.GetFunctionOutput
+	if testAccGetPartition() == "aws-us-gov" {
```

Comment:
Nit: we also have access to the const available from the aws go sdk
```suggestion
	if testAccGetPartition() == endpoints.AwsUsGovPartitionID {
```

Reply:
I copied the lines directly from the code in `main` so I won't changed here.
Thanks.